### ISSUE TITLE
A small change in Spanish translation

### ISSUE THREAD

**Issue by Gadeo** [rejuvenate/lovelace-horizon-card#72]
Hi, thanks for your hard work, congrats. I just got a small (but important) Spanish translate request: is it possible to change in the sun set translation "Atardecer", that is similar to afternoon and is not correct, for "Puesta del Sol" that is sun set? It got more sense. Thanks for your effort.

**Comment by Gadeo** [comment]
Yes, that`s true!! "Orto" and "ocaso" are more technical and beautiful words. If you look for Spanish nautical books It's as it's written, but maybe are lesser-known words and, as you say, "orto" is a bad word for people from South America and the card can be download for people from there. So, I think "Puesta del Sol" can be more "international" :)

**Comment by ThomDietrich** [comment]
@Gadeo just for my own understanding, your translation suggestion is for spanish as spoken in Spain? Do you know whether that's also the better translation for e.g. latin america?

**Comment by Gadeo** [comment]
My suggestion is, for Spanish from Spain: "orto" and "ocaso', from Latin America is must to be the same but, like @avataar said, "orto" is a bad word in Argentina/Chile/Uruguay but not in Mexico/Venezuela/Colombia.So if you have the possibility to translate it in Spanish(Lat) just write 'Salida del Sol' and "Puesta del Sol" that always is better that "Atardecer'

**Comment by avataar** [comment]
Let's stick with a single Spanish translation. I like the "puesta del sol" solution.

**Comment by Gadeo** [comment]
Yes, that's great. Thank you!

**Comment by yvmexcl** [comment]
Hello, thank you for your time and sharing such a beautiful and practical card, for Latin America "Puesta del Sol" is also used, which is when the sun goes down, "Orto", it does not work hahaha greetings from Santiago de Chile

**Comment by ThomDietrich** [comment]
Good call — "Atardecer" really reads as afternoon. Changed it to "Puesta del sol" in #206. Thanks!
